Signal/src/ViewControllers/ConversationView/MessagesViewController.m

```diff
@@ -2693,7 +2702,8 @@ - (void)ensureDynamicInteractions
         [ThreadUtil ensureDynamicInteractionsForThread:self.thread
                                        contactsManager:self.contactsManager
                                        blockingManager:self.blockingManager
-                                          dbConnection:self.editingDatabaseConnection
+                                      readDBConnection:self.uiDatabaseConnection
+                                     writeDBConnection:self.editingDatabaseConnection
```

Comment:
`ensureDynamicInteractionsForThread` should operate on the latest thread contents, so we should _read_ from uiDatabaseConnection and _write_ to `editingDatabaseConnection`.